### PR TITLE
add foundry to an existing hardhat project

### PR DESCRIPTION
**Pull request by fjun99**
Add a 4-step instruction to add foundry to an existing hardhat project in `config/hardhat.md`

### CODE REVIEW

File: src/config/hardhat.md [hunk 1]
```diff
@@ -69,3 +69,49 @@ paths: {
...
+src = 'contracts'
+out = 'out'
+libs = ['node_modules', 'lib']
+test = 'test/foundry'
+cache_path  = 'forge-cache'
+
+# See more config options https://book.getfoundry.sh/reference/config.html
+```
+
+3. Create a `remappings.txt` to make Foundry project work well with VS Code Solidity extension:
```
Comment (onbjerg) [inline src/config/hardhat.md:101]: This is in a guide before the Hardhat guide, so no need here imo

Comment (fjun99) [reply src/config/hardhat.md:101]: I think this is helpful for those who want to migrate to foundry from hardhat. And I was one of them several days ago.

Comment (fjun99) [reply src/config/hardhat.md:101]: > This is in a guide before the Hardhat guide, so no need here imo

I think this is helpful for those who want to migrate to foundry from hardhat. And I was one of them several days ago.

Comment (onbjerg) [reply src/config/hardhat.md:101]: I don't have strong feelings about it, but that information is in a separate guide and is VSCode-specific. Not everyone who uses Hardhat uses VSCode. Going to merge this but I'd still argue it's better of being in it's own guide :smile: Thanks for the contribution!

**Review by onbjerg — APPROVED** [review]

File: src/config/hardhat.md [hunk 2]
```diff
@@ -69,3 +69,49 @@ paths: {
...
+
+```
```
Comment (onbjerg) [inline src/config/hardhat.md:103]:
````suggestion
```ignore
````

File: src/config/hardhat.md [hunk 3]
```diff
@@ -69,3 +69,49 @@ paths: {
...
+ds-test/=lib/forge-std/lib/ds-test/src/
+forge-std/=lib/forge-std/src/
+```
+
+See more on `remappings.txt` and VS Code Solidity extension: [Remapping dependencies](projects/dependencies.html?highlight=remap#remapping-dependencies), [Integrating with VSCode](/config/vscode.html)
+
+4. Make a sub-directory `test/foundry` and write Foundry tests in it. 
+
+Let's put the sample test file `Contract.t.sol` in this directory and run Foundry test
+```
```
Comment (onbjerg) [inline src/config/hardhat.md:113]:
````suggestion
```bash
````

**Review by onbjerg — COMMENTED** [review]
Nits (makes tests pass)

**Comment by onbjerg** [comment]
@all-contributors add @fjun99 for maintenance

**Pull request merged**